issue.go

```diff
@@ -733,6 +733,24 @@ func (s *IssueService) UpdateComment(issueID string, comment *Comment) (*Comment
 	return responseComment, resp, nil
 }
 
+// DeleteComment Deletes a comment from an issueID.
+//
+// JIRA API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-api-3-issue-issueIdOrKey-comment-id-delete
+func (s *IssueService) DeleteComment(issueID, commentID string) error {
+	apiEndpoint := fmt.Sprintf("rest/api/2/issue/%s/comment/%s", issueID, commentID)
+	req, err := s.client.NewRequest("DELETE", apiEndpoint, nil)
+	if err != nil {
+		return err
+	}
+
+	_, err = s.client.Do(req, nil)
```

Comment:
Can you catch the `resp` parameter here and if something goes wrong, include this in the error message?
Many of the other calls are doing this. Check: https://github.com/andygrunwald/go-jira/blob/master/issue.go#L703-L707